packages/amplify_api/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   amplify_api_plugin_interface: 0.2.3
   amplify_core: 0.2.3
+  async: ^2.6.0
```

Comment:
Is this only used in tests? Should it be a dev dependency?

Reply:
Good catch - fixed!